### ISSUE TITLE
Increase logo visibility in contrast themes / high-contrast mode

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail.org#463]
Follow-up to #454. The logos of Wagtail users visible on the site homepage don’t display too well in Windows high-contrast mode / contrast themes. This is particularly problematic for the Royal College of Arts logo, which is fully invisible when displayed in a high-contrast "dark" theme.

I think the ideal fix would be to apply similar styles to the ones used for the logos in the site’s dark theme:

https://github.com/wagtail/wagtail.org/blob/e8d45141c6f1e6bc97b4dead85cb560cfab9710a/wagtailio/static/sass/components/_logos.scss#L61-L68

By adding a `forced-color-adjust: none` alongside the background color, we’d ensure the logos are always displayed with the intended background color. It seems to me like all those styles could be present for all themes rather than forced-colors mode / the dark theme only, so the logos display as consistently as possible.

**Comment by kituuu** [comment]
Can I fix this? But I don't have a windows system to check this.

**Comment by GajulaSamatha** [comment]
Hii sir,
I want to work on this issue, and I need some clarification. Please feel free to clarify.

1) 
![Image](https://github.com/user-attachments/assets/eb03b3a7-a162-4566-942d-faa61434ba3b)


Are you referring to this logo? 

2) Or is it the following one?

![Image](https://github.com/user-attachments/assets/d3459264-4ea2-4f6b-903f-508826a738e5)

Thank you in advance sir.

**Comment by GajulaSamatha** [comment]
Hello sir,

I have now cleared my doubts (as you suggested, the changes should be in logos.scss, not the Wagtail header logo) and attempted to solve the issue. To confirm, I am attaching screenshots of the output:

1) White-Contrast in system with Light Mode in website.

![Image](https://github.com/user-attachments/assets/2b8b7cc0-fa55-416e-b6c4-f7e4363054d6)

2) White-Contrast in system with Dark Mode in website.

![Image](https://github.com/user-attachments/assets/800778f2-681b-4faf-8d0b-4d0824be830d)

3) Dark-Contrast in system with Dark Mode in website.

![Image](https://github.com/user-attachments/assets/e71b3738-56a5-48ea-96c1-36d64664c100)

4) Dark-Contrast in system with Light Mode in website.

![Image](https://github.com/user-attachments/assets/f91d3a99-cbd5-41ee-ad9e-c24302d8b695)

I tested this in Chrome, Microsoft Edge, and Firefox.

Before enabling contrast mode, the website’s current Light/Dark mode affects the visual changes. However, when switching between system Dark/White Contrast settings, the website does not reflect any changes and maintains its own Light/Dark mode.

Could you please confirm if this is the expected behavior? 
I sincerely apologize for the long message. 
If there are any mistakes, please excuse me and kindly guide me in the right direction.

Thank you, sir! 😊

**Comment by thibaudcolas** [comment]
@GajulaSamatha yup, that look like what I’d expect!

**Comment by GajulaSamatha** [comment]
Thank you so much, @thibaudcolas sir, for confirming that my output meets your expectations. I’m really happy to hear that! 😊

Also, sir, I noticed that our Wagtail logo is not visible in the 2nd and 4th pictures. Shall I try to handle those cases as well? I’ll definitely try my best sir.

**Comment by thibaudcolas** [comment]
@GajulaSamatha that’s up to you! That issue is logged separately here: #303.

**Comment by GajulaSamatha** [comment]
Oh ok sir, thank you for your time🙂👍.
